SofaKernel/framework/sofa/core/behavior/ForceField.inl

```diff
@@ -83,7 +83,7 @@ void ForceField<DataTypes>::addDForce(const MechanicalParams* mparams, MultiVecD
 
 #ifndef NDEBUG
         if (!mparams->getKFactorUsed())
-            serr << "WARNING " << getClassName() << " (in ForceField<DataTypes>::addDForce): please use mparams->kFactor() in addDForce" << sendl;
+            msg_error() << "WARNING " << getClassName() << " (in ForceField<DataTypes>::addDForce): please use mparams->kFactor() in addDForce";
```

Comment:
Thanks a lot, and congrats for your very first PR @VLinternship !

Concerning the messaging API in SOFA, "warning" messages should be reported using msg_warning, instead of msg_error. 
Also, msg_error / msg_warning and msg_info automatically write [ERROR],[WARNING],[INFO] respectively at the beginning of the given message, so :
```cpp
            msg_warning() << "WARNING " << getClassName() << [...];
```
can become 
```cpp
            msg_warning() <<  getClassName() << [...];
```
Good work for sure, anyway! :+1: